test/resource-timing.spec.ts

```diff
@@ -16,20 +16,24 @@
  */
 
 import { expect, it } from './fixtures';
+import * as os from 'os';
 
-it('should work', async ({ page, server }) => {
+it('should work', (test, { browserName, platform }) => {
+  const isBigSur = platform === 'darwin' && parseInt(os.release(), 10) >= 20;
+  test.fail(isBigSur && browserName === 'webkit', 'Resource timing is -1 on BigSur');
+}, async ({ page, server }) => {
   const [request] = await Promise.all([
     page.waitForEvent('requestfinished'),
     page.goto(server.EMPTY_PAGE)
   ]);
   const timing = request.timing();
-  expect(timing.domainLookupStart).toBeGreaterThanOrEqual(0);
+  expect(timing.domainLookupStart).toBeGreaterThanOrEqual(-1);
```

Comment:
Why do we mark it as failing but also change the expectation?

Reply:
By a mistake...